include/serialization_utils/cereal_cvmat.hpp

```diff
@@ -60,7 +60,7 @@ void save(Archive& archive, const cv::Mat& mat)
             archive(row_data);
         }
     }
-};
+}
```

Comment:
Thanks for taking care of this!  I wanted to fix this warning for quite a while but never managed to actually do it :D